### PR TITLE
add secrets for S3

### PR DESCRIPTION
**Pull request by mattcasey**
I also want to replace 'stg' with 'prd' as default. The last two devs that had to add variables were confused that we use /prd/ in ebextensions_cron and asked me where to set prd for the webapp.

### CODE REVIEW

File: .ebextensions/00_env_vars.config [hunk 1]
```diff
@@ -4,23 +4,27 @@
...
-        COMPOSE_PROFILES: "stg"
-        IMGTAG: ""
\ No newline at end of file
+        NODE_ENV: "prod"
+        NPM_SCRIPT: "start:prod"
+        COMPOSE_PROJECT_NAME: "prd"
+        COMPOSE_PROFILES: "prd"
+        IMGTAG: ""
+        S3_UPLOAD_BUCKET: "{{pull:secretsmanager:/io.cv.app/prd/aws/s3:SecretString:bucket}}"
+        S3_UPLOAD_KEY: "{{pull:secretsmanager:/io.cv.app/prd/aws/s3:SecretString:access_key}}"
```
Comment (yiwork) [inline .ebextensions/00_env_vars.config:28]: you don't need this if you have instances have IAM profile that permits access to the bucket

File: .ebextensions/00_env_vars.config [hunk 2]
```diff
@@ -4,23 +4,27 @@
...
+        S3_UPLOAD_REGION: "{{pull:secretsmanager:/io.cv.app/prd/aws/s3:SecretString:region}}"
+        S3_UPLOAD_SECRET: "{{pull:secretsmanager:/io.cv.app/prd/aws/s3:SecretString:access_key_secret}}"
```
Comment (yiwork) [inline .ebextensions/00_env_vars.config:30]: Ditto point above

**Review by yiwork — CHANGES_REQUESTED** [review]
remove the s3 access keys unless this is a bucket we don't own

**Comment by mattcasey** [comment]
We decided offline that using client key/secret is fine for now, Connie will see if she can get generating the federated token to work using instance roles instead

**Pull request merged**